app/routers/system.py

```diff
@@ -56,12 +56,16 @@ async def login(i: LoginInput):
             )
             data = parse_key_value_text(result)
             if data["correct"] == "1":
-                return sign_jwt()
+                token = sign_jwt().get("access_token")
+                response.set_cookie("access_token", token)
+                return token
```

Comment:
Last nitpick: return the whole object from sign_jwt() instead of only the token to not break existing applications.

Reply:
Done.